### DISCUSSION TITLE
如何减少候选词中词组的数量？

### DISCUSSION THREAD

**Discussion by HyperBioGhost** [rime/home#829] [Q&A]
RT。
输入多个词组时，会遇到提示的候选词组不包含希望输入的词，或者说希望输入某些单个的字，恰好组合起来能拼成另外一个词，此时因为候选的词组过多，需要一页一页翻过去，翻到所有的候选词组都结束之后，才会显示单个的字，然后再在这些单个的字当中寻找自己需要的。
这样会导致在输入时花费较多时间翻页，把所有不需要的词组一页一页翻过去……
有没有什么办法可以限制候选词组的数量，比如我翻过去两页之后，就不提供候选词组，强制出现单个的字。
这个问题似乎比较难描述，搜不到有效的解决办法，在这儿提交一个issue希望能找到靠谱的解决方案。
感谢！

**Comment by lotem** [comment]
目前沒有這樣的過濾機制。
一般認為“詞組”是可有可無的候選項，因為即使在詞典中缺失，也可以通過單字組合出來。
但是Rime的程序主要按音節數來界定基本元素和組合元素，而不是漢字數。
假如“花兒”將兒化音拼作一個音節 huar，那麼這個詞在這個讀音下無法用其他單字的拼音組合出來，就不能等同於一般意義上的詞組。

手動按音節篩選的方法是按<kbd>Tab</kbd>或方向鍵聚焦第一個音節的編碼再選擇對應的單字。

**Reply by blackhole889** [reply]
这个想法很好，我早有此意，以前拼音加加就有这个设置。但在rime里实现比较难。
